specs/market-api.yaml

```diff
@@ -1065,7 +1065,13 @@ components:
           schema:
             type: array
             items:
-              $ref: '#/components/schemas/AgreementEvent'
+              anyOf:
```

Comment:
Correct it to `AgreementOperationEvent` and ensure that `agreementId` will be present in the event. Remove them from the sub-classes.